R/descriptives.R

```diff
@@ -271,7 +271,9 @@ Descriptives <- function(jaspResults, dataset, options) {
   if (options$standardErrorMean)    stats$addColumnInfo(name="Std. Error of Mean",          title=gettext("Std. Error of Mean"),      type="number")
   if (options$median)               stats$addColumnInfo(name="Median",                      title=gettext("Median"),                  type="number")
   if (options$mode)                 stats$addColumnInfo(name="Mode",                        title=gettext("Mode"),                    type="number")
-  if (options$standardDeviation)    stats$addColumnInfo(name="Std. Deviation",              title=gettext("Std. Deviation"),          type="number")
+  if (options$standardDeviation)    stats$addColumnInfo(name="Std. Deviation",              title=gettext("Std. Deviation"),   
+  type="number")
+  if (options$cOfVariation)      stats$addColumnInfo(name="Coefficient of Variation",    title=gettext("Coefficient of Variation"),    type="number")
```

Comment:
So you slightly messed up the alignment here:
```suggestion
  if (options$standardDeviation)    stats$addColumnInfo(name="Std. Deviation",              title=gettext("Std. Deviation"),          type="number")
  if (options$cOfVariation)         stats$addColumnInfo(name="Coefficient of Variation",    title=gettext("Coefficient of Variation"),type="number")
```
GitHub probably won't show this nicely but it should look somewhat like this:
![image](https://user-images.githubusercontent.com/21319932/107749554-12176600-6d1b-11eb-9d79-e41e79997cf4.png)

Reply:
I can see the messed up alignment in kate. I do not know how I missed it. Maybe I also used editor on windows on the other machine. Will stick to kate now. It recognizes R too.